CMakeLists.txt

```diff
@@ -166,3 +158,36 @@ add_library(onut STATIC
     src/zlib/uncompr.c
     src/zlib/zutil.c
     )
+
+target_include_directories(onut
+    PUBLIC
+        /opt/vc/include/
+        /opt/vc/include/interface/vcos/pthreads
+        /opt/vc/include/interface/vmcs_host/linux 
+        ./include 
+    PRIVATE
+        ./src
+)
```

Comment:
in case you were wondering what this `PUBLIC` and `PRIVATE` means: your sample now only links `onut` and nothing else. Anything `PUBLIC` will automatically call  `include_directories` for every projects using `onut`, so you do not have to repeat these paths in your samples. cmake will add them for you. `PRIVATE` means the opposite. it won't include these paths to the dependant projects.